docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+
+Complete the form to create the first Administrator user, then click **Let's start**.
+
+You now have access to the [admin panel](http://localhost:1337/admin):
+
+![Admin panel screenshot glimpse](../assets/quick-start-guide//qsg-handson-part1-01-admin_panel.png)
+
+:::warning 🥳 CONGRATULATIONS!
+You have just created a new Strapi project! You can start playing with Strapi and discover the product by yourself using our [User Guide](/user-docs/latest/getting-started/introduction.md), or proceed to part 2 below.
 :::
 
-::::
+## 🛠 Part 2: Build your content
+
+The installation script has just created an empty project. We will now guide you through creating a restaurants directory, based on our [FoodAdvisor](https://github.com/strapi/foodadvisor) example application.
+
+The admin panel of Strapi runs at [http://localhost:1337/admin](http://localhost:1337/admin). This is where you will spend most of your time creating and updating content.
+
+:::tip TIP
+If the server is not already running, in your terminal, `cd` into the `my-project` folder and run `npm run develop` (or `yarn develop`) to launch it.
+:::
+
+### Step 1: Create collection types with the Content-Types Builder
+
+#### Create a `Restaurant` collection type
+
+1. Go to [_Plugins > Content-Types Builder_](http://localhost:1337/admin/plugins/content-type-builder) in the left-hand menu.
+2. Click on **+ Create new collection type**.
+3. Type `restaurant` for the **Display name**, and click **Continue**.  
+4. Click the **Text** field, then type `name` in the **Name** field.
+5. Switch to the **Advanced Settings** tab, and check the **Required field** and the **Unique field** settings.
+6. Click on **+ Add another Field**.
+7. Choose the **Rich Text** field.
+8. Type `description` under the **Name** field, then click **Finish**.
+9. Finally, click **Save** and wait for Strapi to restart.
 
-## 2. Create an Administrator user
+![GIF: Create Restaurant collection type in Content-Types Builder](../assets/quick-start-guide/qsg-handson-restaurant.gif)
 
-Navigate to [**http://localhost:1337/admin**](http://localhost:1337/admin).
+#### Create a `Category` collection type
 
-Complete the form to create the first **Administrator** user.
+1. Go to [_Plugins > Content-Types Builder_](http://localhost:1337/admin/plugins/content-type-builder) in the left-hand menu.
+2. Click on **+ Create new collection type**.
+3. Type `category` for the **Display name**, and click **Continue**.
+4. Click the **Text** field, then type `name` in the **Name** field.
+5. Switch to the **Advanced Settings** tab, and check the **Required field** and the **Unique field** settings.
+6. Click on **+ Add another Field**.
+7. Choose the **Relation** field.
+8. On the right side, click the **Category** dropdown and select **Restaurant**. 
+9. In the center, select the icon that represents `many-to-many`. The text should read `Categories has and belongs to many Restaurants`.
```

Comment:
I would add `that represents a many-to-many relation.` and adding the <img width="27" alt="Screen Shot 2021-05-29 at 13 19 43" src="https://user-images.githubusercontent.com/5716596/120068357-913d6980-c080-11eb-945d-356f7134c6e2.png"> directly in the text line.
Like we are doing in the User Guide to talk about icons.
I think that could help to make it clearer. Even if the screenshot is really clear.

Reply:
Great idea, thanks Jim! I'll get inspiration from what Mégane did in the user guide.